### ISSUE TITLE
Errors with PHP 8.1

### ISSUE THREAD

**Issue by luis-j** [mkucej/i-librarian-free#68]
Hi,

I'm on Arch Linux, and after upgrading php from 8.0.14 to 8.1.2, this error pops up all the time:

500 Internal Server Error
E_DEPRECATED: htmlspecialchars(): Passing null to parameter #1 ($string) of type string is deprecated.

**Comment by SoulEater45** [comment]
I am having similar problems when trying to import using an ISBN. I am getting the error message:
```
500 error
E_NOTICE: Trying to access array offset on value of type bool.
```
The console of chrome and firefox says:
```
Failed to load resource: the server responded with a status of 500 ()
```
for the endpoint `/index.php/ol/convert:1`.

**Comment by luis-j** [comment]
Anyone with enough knowledge of PHP to solve this?
Now even Ubuntu switched to php 8.1 and as soon as people begin upgrading their systems, I, Librarian will stop working.

**Comment by mkucej** [comment]
We'll be adjusting to PHP 8.1 soon.

**Comment by luis-j** [comment]
That's great, Martin, thank you!

**Comment by mkucej** [comment]
fixed in 5.10.1
